src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import TalentStatisticBox from 'interface/others/TalentStatisticBox';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import Events from 'parser/core/Events';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+
+/**
+ * Example Report: https://www.warcraftlogs.com/reports/GdDhNAZgp2X3PbVC#fight=31&type=summary&source=10
+ */
+const MAX_FURY = 120;
+class BlindFury extends Analyzer{
+
+  gained = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.BLIND_FURY_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.EYE_BEAM), this.onEyeBeamsCast);
+  }
+
+  //Since this will always max out your fury, we only care how much did you gain.
+  onEyeBeamsCast(event) {
+    this.gained += (MAX_FURY - (event.classResources[0].amount - event.classResources[0].cost));
```

Comment:
Would it be useful to count the wasted still?
Casting EB while being nearly capped is still bad right?

Reply:
Reached out to the guide writer on discord to see what he would consider a good cut off.

Reply:
Do you want to merge it now anyway or wait on feedback?

Reply:
Luckily he replied quickly and said the cut off is 50 fury. So will be adding that in.